js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillOpJobPartitionsTable.tsx

```diff
@@ -0,0 +1,80 @@
+import {Box, NonIdealState, Table} from '@dagster-io/ui-components';
+import {useMemo} from 'react';
+
+import {StatusBar} from './BackfillAssetPartitionsTable';
+import {BackfillTarget} from './BackfillRow';
+import {BackfillDetailsBackfillFragment} from './types/useBackfillDetailsQuery.types';
+import {failedStatuses, inProgressStatuses, successStatuses} from '../../runs/RunStatuses';
+import {numberFormatter} from '../../ui/formatters';
+import {buildRepoAddress} from '../../workspace/buildRepoAddress';
+
+export const BackfillOpJobPartitionsTable = ({
+  backfill,
+}: {
+  backfill: BackfillDetailsBackfillFragment;
+}) => {
+  const results = useMemo(() => backfill.partitionStatuses?.results || [], [backfill]);
+
+  const repoAddress = backfill.partitionSet
+    ? buildRepoAddress(
+        backfill.partitionSet.repositoryOrigin.repositoryName,
+        backfill.partitionSet.repositoryOrigin.repositoryLocationName,
+      )
+    : null;
+
+  const inProgress = useMemo(
+    () => results.filter((result) => inProgressStatuses.has(result.runStatus as any)),
```

Comment:
Is there a better option than `any` on these?

Reply:
the problem is that `runStatus` could be `null` and `inProgressStatuses` doesn't accept null :(